ui/helpers/job.js

```diff
@@ -237,15 +237,6 @@ export const addAggregateFields = function addAggregateFields(job) {
   return job;
 };
 
-// Remove any fields we don't need for display in the greater job field after
```

Comment:
This was the troublesome part that caused retriggers and add-new-jobs to fail.  We need the ``signature`` and ``job_type_name`` fields on jobs.